docs/maven-tutorial.md

```diff
@@ -0,0 +1,144 @@
+capability, read
+[the Nexus Maven documentation](https://books.sonatype.com/nexus-book/reference/config.html)
+to configure your build properly.
+
+## Setting up Nexus
+
+Use `new-app` to download and deploy the official Nexus Docker image.
+```bash
+oc new-app sonatype/nexus
+```
+
+After the build has finished, run:
+```bash
+oc get svc
+```
+
+The output should resemble:
+```bash
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+```
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
+
+> NOTE: Nexus comes pre-configured for the Central Repository, but you may need
+> others for your application. At this point, add any special repositories to
+> Nexus.
+
+#### Using Probes to Check for Success
+
+This is a good time to set up readiness and livenes probes. These will
+periodically check to see that Nexus is running properly. More information on
+probes can be found
+[here](https://docs.openshift.org/latest/dev_guide/application_health.html).
+
+```bash
+oc set probe dc/nexus \
+	--liveness \
+	--failure-threshold 3 \
+	--initial-delay-second 30 \
+	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://nexus:8081/content/groups/public
+```
+
+#### Adding Persistence to Nexus
+
+> NOTE: If you don't want persistent storage, continue to
+> [Connecting to Nexus](#connecting-to-nexus). However, your cached
+> dependencies will be lost with the pod.
+
+Now we'll create a Persistent Volume Claim for Nexus, so that our dependencies
+will not be lost when the Pod running the server terminates. If you do not have
+admin access on your cluster, ask your system administrator to create a
+Read/Write Persistent Volume for you.  Otherwise, read
+[Managing Volumes in OpenShift](https://docs.openshift.org/latest/dev_guide/volumes.html)
+for instructions on creating a persistent volume.
+
+Add a Persistent Volume Claim to the Nexus Deployment Config.
+```bash
+oc volumes dc/nexus --remove --confirm
+oc volumes dc/nexus --add \
+	--name 'nexus-pvc' \
+	--type 'pvc' \
+	--mount-path '/sonatype-work/' \
+	--claim-name 'nexus-pv' \
+	--claim-size '1G'
+```
+
+This removes the previous volume for the Deployment Config, and adds a claim
+for 1 gigabyte of storage mounted at `/sonatype-work`, which is where our
+dependencies will be stored. Due to the change in config, the Nexus image will
+be redeployed automatically.
+
+Verify that Nexus is running by refreshing the Nexus page in your browser. You
+can monitor the builds progress using
+```bash
+oc get pods -w
+```
+
+## Connecting to Nexus
+
+The next steps will be pointing the build process to use our Nexus repository.
+The rest of the tutorial will be using
+[this sample repository](https://github.com/bparees/openshift-jee-sample.git),
```

Comment:
we desperately need to move this into something like github.com/openshift/jee-ex, using my personal repo in official docs is bad (though you are not the first).

Can you work with @danmcp to get that repo created and move the content over there?